src/fiatExchanges/SelectProvider.tsx

```diff
@@ -230,6 +241,17 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
     )
   }
 
+  const analyticsData = getProviderSelectionAnalyticsData({
+    normalizedQuotes,
+    legacyMobileMoneyProviders,
+    exchangeRates,
+    tokenInfo,
+    centralizedExchanges: exchanges,
+    coinbasePayAvailable: coinbasePayVisible,
+    transferCryptoAmount: cryptoAmount.toFixed(2),
```

Comment:
The .toFixed(2) would probably be more appropriate inside `getProviderSelectionAnalyticsData` for consistency